notifications.bs

```diff
@@ -426,6 +456,8 @@ interface Notification : EventTarget {
   static readonly attribute NotificationPermission permission;
   [Exposed=Window] static void requestPermission(optional NotificationPermissionCallback callback);
 
+  static readonly attribute unsigned long maxActions;
```

Comment:
Historically, the DAP WG had fingerprinting concerns around exposing details of hardware vibrators.

By contrast, number of actions supported has much less fingerprinting risk, since it's only based on screen size and possibly OS, both of which are already exposed. Indeed Blink currently intends to always support 2 actions (both on Android and desktop).

Also, our thinking on fingerprinting has evolved, based on [data showing that it's infeasible to prevent fingerprinting](https://www.chromium.org/Home/chromium-security/security-faq#TOC-Why-isn-t-passive-browser-fingerprinting-including-passive-cookies-in-Chrome-s-threat-model-).